** POD in backup-manager-purge has been cleaned
   Needs to be applied upstream.

   Provided by: Alexis Sukrieh <sukria@sukria.net>


--- a/backup-manager-purge
+++ b/backup-manager-purge
@@ -49,8 +49,12 @@
 Specify the time to live (in days) for the archives. Any archive that is older
 than I<ttl> days will be outdated.
 
+=back
+
 =head1 OPTIONAL ARGS
 
+=over 4
+
 =item B<--files-from=>file
 
 A file containing a list of archives to parse, one archive per line.
